src/module_library/carbon_assimilation_to_biomass.h

```diff
@@ -0,0 +1,198 @@
+#ifndef STANDARDBML_CARBON_ASSIMILATION_TO_BIOMASS_H
+#define STANDARDBML_CARBON_ASSIMILATION_TO_BIOMASS_H
+
+#include "../framework/module.h"
+#include "../framework/state_map.h"
+
+namespace standardBML
+{
+/**
+ * @class carbon_assimilation_to_biomass
+ *
+ * @brief
+ *
+ * ### Overivew
+ *
+ * Models of photosynthesis compute the rate of CO2 assimilation.
+ * However, many BioCro modules require the rate at which photosynthesis
+ * increases plant biomass. Unless a model computes different components
+ * of plant biomass explicitly, the calculation is assumed to be a simple
+ * conversion factor.
+ *
+ * ### How the conversion is computed
+ *
+ * The conversion is based on the fact that plant
+ * biomass is primarily composed of carbohydrates and the ratio of carbon
+ * to oxygen and hydrogen is nearly the same for all carbohydrates. Thus,
+ * given a number of carbon atoms (i.e., moles of carbon), we can estimate
+ * the carbon-hydrogen-oxygen mass (the CHO mass) which will account for
+ * ~95% of plant biomass. We call this ratio \f$\kappa\f$ the CHO carbon
+ * molar mass. This is not to be confused with the molar mass of carbon.
+ *
+ * \f[
+ *  \kappa = \frac{\mu}{(\text{carbon atoms per mol})}
+ * \f]
+ *
+ * Where \f$\mu\f$ is the molar mass. Given the molar mass of a substance,
+ * and the number of carbon atoms contained in the substance, we can compute
+ * \f$\kappa\f$.
+ *
+ * For example, glucose has a molar mass of 180 g/mol and 6 carbon atoms,
+ * assuming isotopes 1-H, 12-C, 16-O. The molar mass varies with the ratio of
+ * isotopes, but the average isotopic ratio is close to the same molar mass.
+ * So every mol C contributes 12 g of C + 2 g H + 16 g O = 30 g CHO mass.
+ * Or (180 g / mol glucose) / (6 mol C / mol glucose) = 30 g CHO / mol C.
+ * All hexoses (e.g., glucose, fructose, galactose) all have the same molecular
+ * formula C6H12O6, and thus the same molar mass (180 g / mol) and all consist
+ * of 6 carbons, again ignoring any variation in isotope distribution.
+ * Thus, they all have a carbon molar mass of about 30 g / mol this value.
```

Comment:
"Thus, they all have a carbon molar mass of about 30 g / mol this value." -> "Thus, they all have a carbon molar mass of about 30 g / mol."